src/planet-a/PlanetGeometryA.cpp

```diff
@@ -66,14 +77,16 @@ void PlanetGeometryA::initialize(float radius)
 
     // calculate total max vertices
     m_vrtxMax = m_vrtxSharedMax + m_chunkMax * (m_vrtxPerChunk
-            - m_vrtxSharedPerChunk);
+                                                - m_vrtxSharedPerChunk);
 
     // allocate vectors for dealing with chunks
     m_indxBuffer.resize(m_chunkMax * m_indxPerChunk * 3);
     m_vrtxBuffer.resize(m_vrtxMax * m_vrtxSize);
     m_chunkToTri.resize(m_chunkMax);
     m_vrtxSharedUsers.resize(m_vrtxSharedMax);
-
+    m_vrtxSharedIcoCorners.resize(m_icoTree->get_vertex_buffer().size()
```

Comment:
This is allocating the space, and then also initializing. Is that your intention?

Reply:
Initializing them to 0 means they're referring to the first vertex, so they're initialized to gc_invalidVrtx. This vector is intended to be parallel with IcoSphereTree::m_vrtxBuffer